collect_app/src/main/java/org/odk/collect/android/formentry/audit/AuditEventSaveTask.java

```diff
@@ -65,8 +65,6 @@ protected Void doInBackground(AuditEvent... params) {
             try {
                 if (fw != null) {
```

Comment:
It looks like it's never possible and that's why I got rid of those additional logs but I didn't have enough bravery to get rid of the null checks too. 
`fw` is null until we call `fw = new FileWriter(file, true);` and `FileWriter` can throw `IOException` there so in such a case it might be null right? Rare but possible.

Reply:
I had missed that `FileWriter` can throw an `IOException`!